tests/flows/teleporter_registry.go

```diff
@@ -76,8 +75,9 @@ func TeleporterRegistry(network interfaces.LocalNetwork) {
 	utils.SetChainConfig(chainConfigs, subnetAInfo, warpEnabledChainConfigA)
 
 	// Restart nodes with new chain config
-	nodeNames := network.GetAllNodeNames()
-	network.RestartNodes(ctx, nodeNames, runner_sdk.WithChainConfigs(chainConfigs))
+	nodeIDs := network.GetAllNodeIDs()
+	network.SetChainConfigs(chainConfigs)
```

Comment:
For my own understanding, what's the difference between this call to `network.SetChainConfigs` and the calls to `utils.SetChainConfig` above? Can we consolidate them/remove one of them, or explain why both are needed in a comment here?

Reply:
I did a tiny refactor on the `utils` one to make it more self-explanatory, in https://github.com/ava-labs/teleporter/pull/401/commits/eb248d9dd51bf6deebba45698e6714a045e9b56b. let me know what you think